Rakefile

```diff
@@ -88,7 +88,8 @@ task :docs do
   #       text: example body
   @functions = funcs.map do |func|
     func['text'] = func['docstring']['text']
-
+    # TODO: remove once puppet-strings releases fix for :'file::read'
+    func['name'] = func['name'][1..-1].gsub("'", '') if func['name'].start_with?(':')
```

Comment:
If this had not affected ordering I would be more inclined to do it. so i think pinning makes sense. https://github.com/puppetlabs/bolt/pull/1062